### PR TITLE
Kritz

### PR DESCRIPTION
**Pull request by ynot01**
# General Documentation

### Intent of your Pull Request

adds crits medigun (name not yet determined open to suggestions)
- [ ] make it work
- [ ] put it on nukies

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
uber medigun but with critical hits (3x damage)

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
32-64 second charge time

# Changelog

:cl:  
rscadd: Added a critical hit boost medigun
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
Fucken pocket medics stomping pubs with their kritz

**Comment by patpol4** [comment]
Rebase to tf2 when

**Comment by ynot01** [comment]
im sorry it s bbad
